tests/common-lib/index.js

```diff
@@ -1,60 +1,6 @@
-// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
 
 const { execSync } = require('child_process')
-const lodash = require('lodash')
-
```

Comment:
### Functions moved to `searchClient.js`
- formatFilters
- formatResourcesFromSearch

### Functions deleted
- closeMatch
- matchPerc
- getMismatchResources